ankh/main.go

```diff
@@ -222,14 +224,14 @@ func execute(ctx *ankh.ExecutionContext) {
 
 func main() {
 	app := cli.App("ankh", "Another Kubernetes Helper")
-	app.Spec = "[-v] [--config] [--kubeconfig] [--datadir] [--context] [--set...] [--filter]"
+	app.Spec = "[-v] [--ankhconfig] [--kubeconfig] [--datadir] [--context] [--set...] [--filter]"
 
 	var (
 		verbose    = app.BoolOpt("v verbose", false, "Verbose debug mode")
 		ankhconfig = app.String(cli.StringOpt{
-			Name:   "config",
+			Name:   "ankhconfig",
 			Value:  path.Join(os.Getenv("HOME"), ".ankh", "config"),
-			Desc:   "The ankh config to use",
+			Desc:   "The ankh config to use. ANKHCONFIG may be set to include a list of ankh configs to merge. Similar behavior to kubectl's KUBECONFIG.",
```

Comment:
What's the use case for allowing multiple ankh config files?

Reply:
i'm thinking that operators will be given references to configs from cluster admins/owners. this makes the most sense when used with an http-based or k8s-based config reference